### ISSUE TITLE
Add certificate SHA256 fingerprint to the current certificate view. Perhaps also sha1 and md5.

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/android#1025]
In #413 and #8 it was suggested and also partially fixed to show the server's certificate.

Example output (current version 1.7.x)
```
5891524778990fb6b40e3d988260e80192975a22c5529379f8d89d072b2b44ac
145a9f612e1f7e77a768ccc7be39b3508756a510ef8b61c7ac38133dc73f8ab4
...
0f0527b439230c9c5b86c6a3b6d4a77ff61df10f4e2a523ea31d820036a8c9cd
b8c70908110b0fea9fe2046feba68b7cb9893ea9825425c66eaf9f6a7b140130
```

However, it is very difficult to verify the whole certificate and a very unusual method.

Therefore I suggest to calculate and display 

### the sha256 fingerprint

and, yes, these are **deprecated** checksums, for user convenience to show also **sha1** and **md5** checksum of the certificate.

Example:

```
sha256  c5:46:51:a8:14:24:dd:ee:64:86:37:7e:d3:a3:a2:5f:51:ca:00:7e:66:74:4c:cd:cd:27:27:78:02:72:40:79
sha1    48:39:2a:40:1f:ea:b4:2d:79:07:fc:fd:1c:8a:ea:90:b4:55:7d:65
md5     9d:6c:de:e1:ac:6e:da:ec:56:86:1d:b9:27:24:0d:bf
```

**Comment by davivel** [comment]
Seen.

Contritbutions are welcome.

**Comment by purigarcia** [comment]
@tobiasKaminsky , @AndyScherzinger , @LukeOwncloud and whoever feels like, contributions are wellcome here :)

**Comment by Wikinaut** [comment]
I propose this as homework for the Berlin Hacking Space @owncloud Conference 2015 https://conference.owncloud.org/conference/oCC2015/schedule#2015-08-29

**Comment by Wikinaut** [comment]
I am now working on this issue! Coming soon.

**Comment by Wikinaut** [comment]
@davivel I am working on it now. Found the place and fixed it already locally.

**Comment by davivel** [comment]
This was fixed in #1025, goes with the release in progress.

**Comment by davivel** [comment]
This was done in #1184. Ready to go in release in progress.
